### ISSUE TITLE
GDrive files do not open in StackEdit

### ISSUE THREAD

**Issue by grahampcharles** [benweet/stackedit#1688]
When an .md file is opened from GDrive (using the Open With / StackEdit... menu or context menu option), it does not open in StackEdit. Instead, the default "welcome" file is opened. 

* Yes, my Google account is connected to StackEdit
* The Google "workspace" is displayed in StackEdit, albeit with no files
* There appears to be no way from StackEdit to "open" a file from GDrive

## Expected behavior:

That there be some way either from GDrive or StackEdit to open an .md file stored in GDrive.

**Comment by boris-design** [comment]
Agree. When in Google Drive I select "Open with… > StackEdit" It shows StackEdit logo for a moment and then "Safari can't open page" message.

**Comment by amrlsayed** [comment]
I have the same problem, I can not open a file from Google drive.
